lang/python/python3-speedtest-cli/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-speedtest-cli
+PKG_VERSION:=2.1.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=speedtest-cli-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/s/speedtest-cli/
+PKG_HASH:=cf1d386222f94c324e3125ba9a0d187e46d4a13dca08c023bdb9a23096be2e54
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/speedtest-cli-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Matt Martz <matt@sivel.net>
+PKG_LICENSE:=Apache2.0
```

Comment:
This is wrong. Please take a look at website for [SPDX License Identifiers](https://spdx.org/licenses/).